game_clean/copy/hl2.bat

```diff
@@ -1,2 +1,2 @@
-@echo off
-@start hl2.exe -steam -game tf -insecure %*
+@ECHO OFF
+START hl2.exe -steam -insecure -novid -nojoy -nosteamcontroller -nohltv -nostartupsound %*

```

Comment:
This file is supposed to provide the set of "required" launch flags.

Reply:
Is `-particle_fallback 2` required?

Reply:
I don't think it was in this file, so no.

Reply:
Oops. I meant in start_tf2.bat